screens/LoginSignUp.js

```diff
@@ -0,0 +1,188 @@
+import React, {useState} from 'react';
+import {KeyboardAwareScrollView} from 'react-native-keyboard-aware-scroll-view';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  TouchableOpacity,
+  TouchableHighlight,
+  TextInput,
+  Linking,
+  KeyboardAvoidingView,
+} from 'react-native';
+
+const LoginSignUp = props => {
+  const [isLogin, setVisible] = useState(true);
+  return (
+    <KeyboardAwareScrollView
+      contentContainerStyle={{flexGrow: 1}}
+      style={styles.removePadding}>
+      <View style={styles.appContainer}>
+        <View style={[styles.alignImage, styles.whiteBackground]}>
+          <View style={[styles.whiteBackground, styles.selfCenter]}>
+            <Image source={require('../assets/images/loginImage.png')} />
+          </View>
+        </View>
+
+        <View style={[styles.tabContainer, styles.whiteBackground]}>
+          <TouchableHighlight
+            onPress={() => {
+              setVisible(true);
+            }}
+            underlayColor="white"
+            style={styles.click}>
+            <Text style={[styles.button, styles.textCenter]}>Login</Text>
+          </TouchableHighlight>
+          <TouchableHighlight
+            onPress={() => {
+              setVisible(false);
+            }}
+            underlayColor="white"
+            style={styles.click}>
+            <Text style={[styles.button, styles.textCenter]}>Signup</Text>
+          </TouchableHighlight>
+        </View>
+
+        {isLogin ? (
+          <View style={styles.active} />
+        ) : (
+          <View style={[styles.active, styles.reverse]} />
+        )}
+
+        <View>
+          <View style={styles.spacer}></View>
+
+          <Text style={styles.inputLabel}>Email address</Text>
+          <TextInput style={styles.input} />
+
+          <View style={styles.shortSpacer} />
+
+          <Text style={styles.inputLabel}>Password</Text>
+          <TextInput style={styles.input} />
+
+          <View style={styles.shortSpacer} />
+
+          <Text
+            style={styles.text}
+            onPress={() => Linking.openURL('http://google.com')}>
+            {isLogin ? 'Forgot passcode ?' : ''}
+          </Text>
+        </View>
+
+        <View
+          style={[styles.alignButton, styles.selfCenter]}
+          onPress={() => {}}>
```

Comment:
Why are we using onPress with View. Shouldn't this be a usable property of Pressable or TouchableOpacity